docs/installation_overview.rst

```diff
@@ -0,0 +1,76 @@
+Installation Overview
+=====================
+
+Setting Expectations
+--------------------
+
+Setting up SecureDrop is a multi-step process, where each step builds on the
+steps that come before it. It's important that you treat the installation
+as a complete process, making sure not to skip any portions of the install
+guide or jump ahead to later content.
+
+Once you have all the necessary hardware,
+:doc:`setting up SecureDrop <install>` will take at least a day's work.
+
+We recommend that you set aside at least a week to
+:ref:`complete and test <Deployment>` your setup.
+
+
+Technical Summary
+-----------------
+
+|SecureDrop architecture overview diagram|
+
+SecureDrop is a tool for sources to communicate securely with journalists. The
+SecureDrop application environment consists of three dedicated computers:
```

Comment:
This section is still duplicated between the general overview and the installation overview. I would suggest keeping it only in the installation overview.

Reply:
I went back and forth on this a bit. I tested a version with the duplicate entry removed (leaving the one in the installation overview in place), but it felt like important context was missing. I think this information is critical in the general overview, but likewise, not reminding folks who are starting to step through the installation would make the process more difficult. Ultimately, I reworked this session entirely, taking out several of the duplicated paragraphs, and then re-framing the ones I kept to make sense within the context of an installation, and specifically written for an administrator. I'm curious to know your thoughts about those changes - does this shorter, more targeted version feel better to you? Or do you still think it's worth scraping one of them entirely?